### ISSUE TITLE
Have you tried to train from scratch and use large learning rate?

### ISSUE THREAD

**Issue by ssssholmes** [gaopengcuhk/Stable-Pix2Seq#4]

**Comment by ssssholmes** [comment]
The authors mentioned they use adamw with 0.003 learning rate without imagenet pretrain. I tried it and find it is really hard to train with such config.

**Comment by gaopengcuhk** [comment]
The largest learning rate for stable training in this codebase is 0.001.

**Comment by Sharpiless** [comment]
Thanks for your great work. But the paper says:"We use the stochastic depth (Huang et al., 2016) with a rate of 10% to reduce overfitting." Have you tried to use stochastic depth method in your model?

**Comment by gaopengcuhk** [comment]
I will try stochastic depth in the future.
